src/components/utils/equipment-types.js

```diff
@@ -25,66 +15,50 @@ export const EQUIPMENT_INFOS_TYPES = {
 export const EQUIPMENT_TYPES = {
     SUBSTATION: {
         type: 'SUBSTATION',
```

Comment:
Maybe we can get rid of this nested field now, and just use EQUIPMENT_TYPES.SUBSTATION ?

Reply:
It would be preferable to engage another technical story instead, as we currently have numerous conflicts to resolve with this particular US. Therefore, it is crucial to avoid further complicating the situation.

thank you :)